### PR TITLE
save only the portion of the drawing shown by the range slider. (#625)

### PR DESCRIPTION
**Pull request by i-make-robots**
for #625

Tested on stanford bunny, seems fine.
Tested on a jpeg.  Correct gcode, but loading gcode into the app smashes all colors to black.
Turtles might want to generate a comment after M0 with the hex value of the color.

### CODE REVIEW

File: src/main/java/com/marginallyclever/makelangelo/Makelangelo.java [hunk 1]
```diff
@@ -876,6 +877,8 @@ private void onChangeSlider(ChangeEvent e) {
         myTurtleRenderer.setLast(top);
         labelRangeMin.setText(Integer.toString(bottom));
         labelRangeMax.setText(Integer.toString(top));
+		SaveGCode.setTrimHead(bottom);
```
Comment (coliss86) [inline src/main/java/com/marginallyclever/makelangelo/Makelangelo.java:880]: Settings static values in the a bad pattern. Every time the user ask to save the gcode, a new `SaveGCode` is instanciated, so thoses values can be set in the constructor of even better in the `save()` method

**Review by coliss86 — CHANGES_REQUESTED** [review]
Remove the static values and add units tests please

**Comment by i-make-robots** [comment]
Show me an elegant way to remove the static variable. I did want it, either.

**Comment by coliss86** [comment]
Here is : https://github.com/MarginallyClever/Makelangelo-software/pull/632

**Review by coliss86 — APPROVED** [review]
LGTM

**Pull request merged**